app/views/_controls.html.erb

```diff
@@ -0,0 +1,9 @@
+<nav id="searchbar" class="navbar navbar-default navbar-static-top">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="searchbar-right navbar-right col-sm-7 col-md-6">
+        <%= render partial: 'catalog/search_form' %>
+      </div>
+    </div>
+  </div>
+</nav><!-- /.navbar -->
```

Comment:
when overriding a partial from sufia/CC (I think this is?), what do you think about leaving a comment with path/version it was overridden from (github 'y' url works nicely) and why? Not sure which partials in this PR are ours, and which are overridden.